### PR TITLE
push to usercache fixes

### PR DESCRIPTION
**Pull request by joshzarrabi**
This should work.

Sorry I forgot to submit this earlier I was wondering why the tests didn't work.

### CODE REVIEW

File: emission/analysis/modelling/tour_model/cluster_pipeline.py [hunk 1]
```diff
@@ -105,6 +105,7 @@ def main(uuid=None, old=True):
     data, bins = remove_noise(data, 300, old=old)
     n, labels, data = cluster(data, len(bins), old=old)
     tour_dict = cluster_to_tour_model(data, labels, old=old)
+    print "tour_dict is %s" % tour_dict
```
Comment (shankari) [inline emission/analysis/modelling/tour_model/cluster_pipeline.py]: Can you remove or comment out the printout statements?

File: emission/storage/decorations/tour_model_queries.py [hunk 2]
```diff
@@ -33,17 +33,18 @@ def get_tour_model(user_id):
 def get_common_places(user_id):
     db = edb.get_common_place_db()
     place_jsons = db.find({"user_id" : user_id})
-    return [esdcpq.make_common_place_from_json(place_json) for place_json in place_jsons]
+    return [esdcpq.make_common_place(place_json) for place_json in place_jsons]
```
Comment (shankari) [inline emission/storage/decorations/tour_model_queries.py]: How did the tests pass?

File: emission/analysis/modelling/tour_model/representatives.py [hunk 3]
```diff
@@ -139,6 +139,7 @@ def cluster_dict(self):
             return
         for i in range(self.num_clusters):
             a = {'sections' : self.clusters[i]}
+            print "self.tour_dict[%s] is %s" % (i, a)
```
Comment (shankari) [inline emission/analysis/modelling/tour_model/representatives.py]: Ditto

File: emission/net/usercache/builtin_usercache_handler.py [hunk 4]
```diff
@@ -168,6 +168,7 @@ def storeCommonTripsToCache(self):
         As of now we will run the pipeline on the full set of data and send that up
         """
         tour_model = esdtmpq.get_tour_model(self.user_id)
+        print "tour_model is %s" % tour_model
```
Comment (shankari) [inline emission/net/usercache/builtin_usercache_handler.py]: Ditto

File: emission/storage/decorations/tour_model_queries.py [hunk 5]
```diff
@@ -33,17 +33,18 @@ def get_tour_model(user_id):
...
+    return [esdctq.make_new_common_trip(trip_json) for trip_json in trip_jsons]
 
 ##################################################################################
 
 def make_tour_model_from_raw_user_data(user_id):
-    list_of_cluster_data = eamtmcp.main(user_id)
-    esdcpq.create_places(user_id)
-    esdctq.set_up_trips(user_id)
+    list_of_cluster_data = eamtmcp.main(user_id, False)
+    print "list_of_cluster_data is %s" % list_of_cluster_data
```
Comment (shankari) [inline emission/storage/decorations/tour_model_queries.py]: Ditto

File: emission/storage/decorations/tour_model_queries.py (hunk 2 above)
Comment (joshzarrabi) [inline emission/storage/decorations/tour_model_queries.py]: I failed to test that function I guess.

Comment (shankari) [inline emission/storage/decorations/tour_model_queries.py]: I'm going to merge this now, but I'd appreciate a test for this before you move on to other things. Also, on Friday, maybe you can walk me through the tests that you have...

**Comment by shankari** [comment]
Also, in the future, can you indicate exactly how you tested the code - e.g. I loaded data from file <....>, I ran the intake pipeline and I saw this output.

```
output
```

**Pull request merged**